cfgov/unprocessed/css/atoms/partially-styled-link.less

```diff
@@ -0,0 +1,50 @@
+/* topdoc
+  name: Partially styled link
+  family: cfgov-atoms
+  patterns:
+    - name: Default example
+      markup: |
+          <a class="a-partially-styled-link">
+            <span class="a-partially-styled-link_unstyled">
+              Unstyled text
+            </span>
+            <span class="a-partially-styled-link_styled">
+              Underlined text
+            </span>
+          </a>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          a.a-partially-styled-link
+            span.a-partially-styled-link_unstyled
+            span.a-partially-styled-link_styled
+  tags:
+    - cfgov-atoms
+*/
+
+.a-partially-styled-link {
+  border-bottom-width: 0px;
+  
+  .a-partially-styled-link_unstyled {
+    color: @black;
+    border-bottom: 1px dotted transparent;
+  }
+  
+  .a-partially-styled-link_styled {
+    border-bottom: 1px dotted @link-underline;
```

Comment:
Judging from the snapshot preview, this isn't being honored and is solid rather than dotted. Can you confirm?

![screen shot 2017-04-28 at 10 35 58 am](https://cloud.githubusercontent.com/assets/1280430/25535964/81eae234-2bfe-11e7-9a86-123f1f4e7a21.png)